applications/LinearSolversApplication/custom_decompositions/eigen_dense_bdc_svd_decomposition.h

```diff
@@ -0,0 +1,259 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+#if !defined(KRATOS_EIGEN_DENSE_BDCSVD_H_INCLUDED)
+#define KRATOS_EIGEN_DENSE_BDCSVD_H_INCLUDED
+
+// External includes
+#include <Eigen/SVD>
+
+// Project includes
+#include "includes/define.h"
+#include "includes/kratos_parameters.h"
+#include "utilities/dense_svd_decomposition.h"
+
+namespace Kratos {
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+template<class TDenseSpace>
+class EigenDenseBDCSVD : public DenseSingularValueDecomposition<TDenseSpace>
+{
+public:
+
+    ///@name Type Definitions
+    ///@{
+
+    /// Definition of the shared pointer of the class
+    KRATOS_CLASS_POINTER_DEFINITION(EigenDenseBDCSVD);
+
+    typedef typename TDenseSpace::DataType DataType;
+    typedef typename TDenseSpace::VectorType VectorType;
+    typedef typename TDenseSpace::MatrixType MatrixType;
+
+    using EigenVector = Eigen::Matrix<DataType, Eigen::Dynamic, 1>;
+    using EigenMatrix = Eigen::Matrix<DataType, Eigen::Dynamic, Eigen::Dynamic>;
+    using DecompositionOptions = Eigen::DecompositionOptions;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    EigenDenseBDCSVD() = default;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    static std::string Name()
+    {
+        return "dense_bidiagonal_divide_and_conquer_singular_value_decomposition";
+    }
+
+    void Compute(
+        MatrixType& rInputMatrix,
+        Parameters Settings) override
+    {
+        // Check user defined options
+        Parameters default_settings = Parameters(R"(
+        {
+            "compute_U" : true,
```

Comment:
Please use lowercase

Reply:
It's an acronym of the left singular vector matrix. I'd keep the capital U as this is how this matrix is commonly known in the community.

Reply:
Hm many things in literature have uppercase names and yet we always use lowercase for the json
Consider that this would be super unintuitive and inconsistent from pretty much every other setting in Kratod

Reply:
Done.